docs/docs/concepts/retrieval_unstructured.mdx

```diff
@@ -0,0 +1,160 @@
+# Retrieval for Unstructured Data
+
+## Motivation
+
+Language models (LLMs) are trained on vast but fixed datasets, which limits their ability to access up-to-date or domain-specific information. 
+
+To enhance their performance on specific tasks, we can augment their knowledge using retrieval systems.
+
+Retrieval systems fetch relevant information from external sources, which can then be included in the prompt given to the model. 
+
+Key benefits of using retrieval systems include:
+
+1. Access to recent or private information
+2. Improved accuracy on domain-specific tasks
+3. Reduced hallucination by grounding responses in retrieved facts
+4. Cost-effective alternative to fine-tuning for factual recall
+
+## Un-Structured Data
+
+Vector stores are a type of retrieval system that work with unstructured data. 
+
+Unstructured documents (e.g., raw text, tables, images) are often embedded into a compressed representation (e.g., a vector) that can used directly for search based upon similarity to an input query.
+
+## Query Translation  
+
+Vector-based similarity search can be sensitive to the query used to retrieve similar documents. 
+
+Yet, ideally, a retrieval system can handle a wide range of (user) inputs, from poorly worded questions to complex multi-part queries.
+
+A popular approch is to use an LLM to review and optionally modify the input is the central idea behind query translation.
+
+This serves as a general buffer, optimizing raw user inputs for your retrieval system. 
+
+For example, this can be as simple as extracting keywords or as complex as generating multiple sub-questions for a complex query.
+
+| Name          | When to use | Description                                                                                                                                                                                                                                                                            |
+|---------------|-------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| [Multi-query](/docs/how_to/MultiQueryRetriever/)   | When you need to cover multiple perspectives of a question. | Rewrite the user question from multiple perspectives, retrieve documents for each rewritten question, return the unique documents for all queries. |
+| [Decomposition](https://github.com/langchain-ai/rag-from-scratch/blob/main/rag_from_scratch_5_to_9.ipynb) | When a question can be broken down into smaller subproblems. | Decompose a question into a set of subproblems / questions, which can either be solved sequentially (use the answer from first + retrieval to answer the second) or in parallel (consolidate each answer into final answer).                                                           |
```

Comment:
Could we link to any guides that exist as part of the main documentation?

See list here:
https://python.langchain.com/v0.1/docs/use_cases/query_analysis/#techniques

For example:
https://python.langchain.com/v0.1/docs/use_cases/query_analysis/techniques/decomposition/

We'd need to identify whether these are present in the 0.3 version of the docs, and if not incorporate them, into the main documentation.

Generally, we should not link to any documentation that's outside of the main docs b/c maintenance becomes very difficult + we're losing theming, linking to API refs etc.

The only exception is langgraph documentation, which we'll add more support to for now (and potentially merge in the future)